creator-node/src/routes/files.js

```diff
@@ -247,46 +254,97 @@ module.exports = function (app) {
     // Do not act as a public gateway. Only serve IPFS files that are tracked by this creator node.
     const dirCID = req.params.dirCID
     const filename = req.params.filename
+    const ipfsPath = `${dirCID}/${filename}`
 
     const queryResults = await models.File.findOne({ where: {
-      multihash: dirCID,
-      type: 'dir'
+      dirMultihash: dirCID,
+      // All other file types are valid and can be served through this route.
+      type: { [models.Sequelize.Op.ne]: 'dir' } // Op.ne = notequal
+      fileName: filename
```

Comment:
this code breaks i believe bc line 262 needs a trailing comma

Reply:
thx! fixed